glide.yaml

```diff
@@ -1,21 +1,21 @@
 package: github.com/openshift/library-go
 import:
 - package: k8s.io/apimachinery
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/api
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/apiserver
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/apiextensions-apiserver
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/kube-aggregator
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: k8s.io/client-go
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0-beta.1
 - package: github.com/openshift/api
-  version: master
+  version: prebase-1.16 # this is wrong, it should be prebase-1.16.0-beta.1
```

Comment:
Details, once we're past prebase step we'll merge to master and it'll be already some .0 version ;)